### PR TITLE
New package: gnome-tweaks-3.28.0.

### PR DESCRIPTION
**Pull request by maxice8**
@Gottox seems we have missed gnome-tweak-tool changing into gnome-tweaks, i changed gnome-tweak-tool into a transitive package that depends on gnome-tweaks, users that update gnome-tweak-tool to 3.28.0 will install gnome-tweaks, users that install gnome first time will get only gnome-tweaks.

after sometime we can delete gnome-tweak-tool

### CODE REVIEW

File: srcpkgs/gnome-tweak-tool/template [hunk 1]
```diff
@@ -1,17 +1,10 @@
 # Template file for 'gnome-tweak-tool'
 pkgname=gnome-tweak-tool
```
Comment (Gottox) [inline srcpkgs/gnome-tweak-tool/template]: make this a subpkg of gnome-tweaks.

Comment (maxice8) [reply srcpkgs/gnome-tweak-tool/template]: a meta subpkg ?

Comment (Gottox) [reply srcpkgs/gnome-tweak-tool/template]: See upower. (Please not that we should not use provides/replaces here, as it had some sideeffects for upower)

Comment (Gottox) [reply srcpkgs/gnome-tweak-tool/template]: You can also leave build_style=meta out, it makes no sense for subpackages.

Comment (maxice8) [reply srcpkgs/gnome-tweak-tool/template]: ok done.

**Pull request merged**